src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/validation/ValidationService.kt

```diff
@@ -79,6 +80,7 @@ class ValidationService(
     messages += recallValidationService.validateUnsupportedRecallTypes(booking)
     messages += postCalculationValidationService.validateSDSImposedConsecBetweenTrancheDatesForTrancheTwoPrisoner(booking, calculationResult)
     messages += postCalculationValidationService.validateSHPOContainingSX03Offences(booking, calculationResult)
+    messages += shpoValidationService.validate(booking)
```

Comment:
As you won't care about the releaseDate anymore this can move to happen before calc